shared/actions/chat2/index.tsx

```diff
@@ -3613,6 +3622,12 @@ const maybeChatTabSelected = (action: RouteTreeGen.OnNavChangedPayload) => {
   return false
 }
 
+const updateDraftState = (action: Chat2Gen.DeselectedConversationPayload) =>
+  Chat2Gen.createMetaNeedsUpdating({
```

Comment:
I think just do `createMetaRequestTrusted` with `force` set to true, otherwise I don't think this will go.

Reply:
Ok I'll look tomorrow or you can. When I tested it it always worked. I put some stuff. Click away. It requested and the snippet showed the draft state